app/common/notifications/notify.service.js

```diff
@@ -27,6 +28,14 @@ function Notify(_, $q, $rootScope, $translate, SliderService, ModalService) {
         $translate(message, translateValues).then(showSlider, showSlider);
     }
 
+    function notifyExport(message, translateValues) {
```

Comment:
Could this become more generic, something like notifyProgress, if the only diff between it and notify is the setting of the loading flag?

Reply:
Or it could be notifyLoading if using the loading flag or notifyProgress and change loading to inProgress

Reply:
Done!